guides/common/assembly_foreman-deployment-planning.adoc

```diff
@@ -5,3 +5,5 @@ include::assembly_deployment-path.adoc[leveloffset=+1]
 include::assembly_common-deployment-scenarios.adoc[leveloffset=+1]
 
 include::assembly_provisioning-requirements.adoc[leveloffset=+1]
+
+include::modules/con_security-considerations.adoc[leveloffset=+1]
```

Comment:
I think none of the security features we mention here are supported on Debian: FIPS, fapolicyd and SELinux are all EL only. At least for Foreman.

Reply:
@jherrman Sorry for confusing you, Jirka. Ewoud is more knowledgeable than me in this area. ^^